Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN \
         \
         && apt-get install -y \
         ghc-8.4.3 \
-        mono-devel='5.14.*' \
+        mono-devel='5.16.*' \
```

Comment:
This should probably be a separate commit in a separate PR.

Reply:
You're right, sorry. This was the only way to have it work locally, but then I rushed the commit a bit. :s